config/migrations/20231108114709-new-minister-data.ttl

```diff
@@ -0,0 +1,124 @@
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a06d2> ;
+    ns1:rangorde 8 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "a3e37299-1321-448f-9d30-d1c921f11d7b" ;
+    dcterms:title "Vlaams minister van Mobiliteit en Openbare Werken" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/ccd0aa72-bad7-48a8-b260-fee95f17a8fd> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/4341629b-40e6-4f37-aea3-62f0900a48dd> ;
+    ns1:rangorde 3 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "ccd0aa72-bad7-48a8-b260-fee95f17a8fd" ;
+    dcterms:title "Vlaams minister van Binnenlands Bestuur, Bestuurszaken, Inburgering en Gelijke Kansen" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/d1290ae5-9a25-4e53-924e-dcfd76cf0886> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0697> ;
+    ns1:rangorde 4 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "d1290ae5-9a25-4e53-924e-dcfd76cf0886" ;
+    dcterms:title "Vlaams minister van Onderwijs, Sport, Dierenwelzijn en Vlaamse Rand" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0703> .
+
+<http://themis.vlaanderen.be/id/mandatee/e19a45d5-aaab-4b2e-9ce2-7156219b1999> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0697> ;
+    ns1:rangorde 4 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "e19a45d5-aaab-4b2e-9ce2-7156219b1999" ;
+    dcterms:title "Vlaams minister van Onderwijs, Sport, Dierenwelzijn en Vlaamse Rand" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/f492c848-0c09-4221-a5aa-3faefd9749cb> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0707> ;
+    ns1:rangorde 1 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "f492c848-0c09-4221-a5aa-3faefd9749cb" ;
+    dcterms:title "Minister-president van de Vlaamse Regering" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0702> .
+
+<http://themis.vlaanderen.be/id/mandatee/f6189ea6-eb79-459e-96b9-261eaa960c1e> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907de6670526694a061b> ;
+    ns1:rangorde 2 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "f6189ea6-eb79-459e-96b9-261eaa960c1e" ;
+    dcterms:title "Vlaams minister van Economie, Innovatie, Werk, Sociale economie en Landbouw" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
```

Comment:
Is this correct? "Vlaams minister van Economie, Innovatie, Werk, Sociale economie en Landbouw" is Jo Brouns according to [this](https://www.vlaanderen.be/vlaamse-regering), so rangorde should be 9 not 2.

But this also points to Hilde Crevits (persoon themis links resolved to her). Crevits mandataris title should be "Vlaams minister van Welzijn, Volksgezondheid en Gezin", but then we're still missing Brouns